### PR TITLE
Clarification on Thread Intro

### PR DESCRIPTION
**Pull request by manjugv**
Address https://github.com/openshmem-org/specification/issues/134

### CODE REVIEW

**Review by abouteiller — APPROVED** [review]

**Comment by manjugv** [comment]
@agrippa @minsii @swpoole @jamesaross @tonycurtis Can you please review?

File: content/threads_intro.tex [hunk 1]
```diff
@@ -52,9 +52,8 @@
 In addition, a blocked thread will not prevent the progress of \openshmem calls
 performed on other \acp{PE}.
 
-\item In the \CONST{SHMEM\_THREAD\_MULTIPLE} thread level, all \openshmem calls are thread-safe.
-Any two concurrently running threads may make \openshmem calls and the outcome
-will be as if the calls executed in some order, even if their execution is interleaved.
+\item In the \CONST{SHMEM\_THREAD\_MULTIPLE} thread level, all \openshmem calls
+are thread-safe .i.e., any two concurrently running threads may make \openshmem calls.
```
Comment (agrippa) [inline content/threads_intro.tex]: I think there's some whitespace issues. How about just:

are thread-safe. That is, any two concurrently running threads may make \openshmem calls.

**Review by agrippa — CHANGES_REQUESTED** [review]

**Comment by minsii** [comment]
Approve with Max's patch.

**Review by tonycurtis — APPROVED** [review]

**Comment by agrippa** [comment]
LGTM

**Comment by jamesaross** [comment]
Best to avoid the confusing language. Approved.

**Review by minsii — APPROVED** [review]

**Review by tonycurtis — APPROVED** [review]

**Comment by swpoole** [comment]
Looks good to me.

**Comment by manjugv** [comment]
Thanks committee. We have approvals now and I will merge.

**Pull request merged**